source/content.js

```diff
@@ -299,6 +300,16 @@ function ajaxedPagesHandler() {
 		enableFeature(extendDiffExpander);
 	}
 
+	if (
+		pageDetect.isIssueList() ||
+		pageDetect.isIssueSearch() ||
+		pageDetect.isPR() ||
+		pageDetect.isPRSearch() ||
+		pageDetect.isSingleFile()
+	) {
+		enableFeature(makeHeadersSticky);
```

Comment:
I'll move this to `onDomReady` since the original styles were applied globally anyways. I don't think there's a need to limit this to certain pages.